.github/labeler.yml

```diff
@@ -0,0 +1,48 @@
+GHA:
+  - .github/**/*
+
+config:
+  - charts/registry/config/**/*
+
+docs:
+  - README.md
+  - docs/**/*.md
+
+build:
+  - Makefile
+  - Dockerfile
+  - docker-bake.hcl
+  - dockerfiles/**/*
+
+ci:
+  - .github/**/*
+  - tests
+  - testutil
+
+vendor:
+  - vendor/**/*
```

Comment:
This should probably use the `dependencies` label (and we should remove the `area/vendor` label, but first relabel existing PRs if it's used)

Reply:
Maybe this could be changed to `go.mod` instead of the `vendor` dir 🤔

Reply:
Yes, I was considering adding those, but also thought changing those should also modify the vendor directory, so perhaps not needed.